startup.py

```diff
@@ -7,8 +7,10 @@
 from datetime import datetime
 from pprint import pprint
 from langchain_core._api import deprecated
+from configs.model_config import LLM_DEVICE
 
-import intel_extension_for_pytorch as ipex
+if LLM_DEVICE=="xpu":
```

Comment:
Should we add a check here for LLM_DEVICE or EMBEDDING_DEVICE both equal to xpu?

Reply:
We should import ipex for the case that only one model is on XPU (even we do not verified this case for now